languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,7 @@
+class SimpleCalculator
```

Comment:
Maybe we add the frozen string literal directive?

Reply:
We talked about leaving string literal directive and `freeze` for later episodes 😄